components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.store.ui/src/main/java/org/wso2/carbon/mediator/store/MessageStoreMediator.java

```diff
@@ -43,6 +48,8 @@
 
     private String messageStoreName;
 
+    private SynapsePath messageStoreExp;

```

Comment:
Shall we add a more meaningful name for the variable?

Reply:
Done